db/migrate/20181001131632_add_conversion_host_id_to_miq_request_tasks.rb

```diff
@@ -0,0 +1,44 @@
+class AddConversionHostIdToMiqRequestTasks < ActiveRecord::Migration[5.0]
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options, Hash
+    belongs_to :conversion_host, :class_name => "AddConversionHostIdToMiqRequestTasks::ConversionHost"
+  end
+
+  class ConversionHost < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :miq_request_tasks, :conversion_host_id, :bigint
+
+    MiqRequestTask.where(:type => 'ServiceTemplateTransformationPlanTask').each do |task|
+      host_id = task.options[:transformation_host_id]
+      next unless host_id
+      host = Host.find_by(:id => host_id)
+      next if host.nil?
+      task.conversion_host = ConversionHost.find_or_create_by!(:resource_id => host.id, :resource_type => host.type) do |ch|
```

Comment:
If you add the `belongs_to :resource, :polymorphic => true` to your [`ConversionHost` stub](https://github.com/ManageIQ/manageiq-schema/pull/281/files#diff-c1d34c28716698230c0bdaeb06a6aeeeR8) you should be able to do `find_or_create_by!(:resource => host)`, this will also help [here](https://github.com/ManageIQ/manageiq-schema/pull/281/files#diff-ad3d27814217293149c9a7bbe6240470R28) and [here](https://github.com/ManageIQ/manageiq-schema/pull/281/files#diff-ad3d27814217293149c9a7bbe6240470R21)

Reply:
Thanks. Helpful and good to memorize.